snapcraft/commands/clean.py

```diff
@@ -26,7 +26,8 @@
 Options:
   -h --help             show this help message and exit.
   -s STEP --step=STEP   only clean the specified step and those that depend
-                        upon it.
+                        upon it. STEP can be one of: pull, build, stage, strip,
+                        snap. See 'snapcraft help plugins' to learn more.
```

Comment:
Ah, good addition, thank you! However, these steps are per-part, and `snap` isn't a per-part step, so it's just pull, build, stage, or strip.